src/main/java/frc/robot/Robot.java

```diff
@@ -36,9 +42,93 @@ public void robotInit() {
         hangingSystem = new HangingSystem();
 
         armJointControlCommand = new ArmJointControlCommand(armJointSystem);
-        armJointSystem.setDefaultCommand(armJointControlCommand);
+        armJointSystem.setDefaultCommand(
+                new ParallelCommandGroup(
+                        new SequentialCommandGroup(
+                                new ArmTelescopicReset(armTelescopicSystem),
```

Comment:
if you want a default command for the arm telescopic, use the appropriate system, don't set it on a different subsystem....

same for the others you placed here